### DISCUSSION TITLE
Can downloaded voice models used for TTS with coqui-ai?

### DISCUSSION THREAD

**Discussion by cryptotester** [coqui-ai/TTS#3818] [General Q&A]
Hi, noob question, sorry. I found a website that seems to have lots of existing and community created models at https://voice-models.com/

I downloaded a few, eg obama, trump etc. my question is:

Can I use such, with coqui-ai a downloaded model, as is, for TTS, meaning starting from a written text as input and get the audio as output?

 Seems an obvious use case, but I haven't found yet an easy way to do this:
 - download an already trained voice (of famous actors etc)
 - save the voice in a TTS program and choose it from a list, configure it a bit, pitch etc
 - type sentences and get the result using that voice
 
 Would be cool if you can point me to the right direction to achieve this. Thank you.

**Comment by eginhard** [comment]
You'll need to use each model with the toolkit they were trained with, so you have to check if a model was trained with Coqui.

**Reply by cryptotester** [reply]
Thank you. The follow up question is, is there a place with many voice models ready to download and use with coqui-ai?
Thanks

**Reply by 7Nic** [reply]
@cryptotester I think the easiest way is to use this website you mentioned to download de RVC model. Then, use this version repo (https://github.com/Vali-98/XTTS-RVC-UI) to import the RVC model and use XTTS at the same time.
You'll be able to use XTTS along with RVC (one of the best combinations)
